src/commands/deployWorkspaceProject/DeployWorkspaceProjectConfirmStep.ts

```diff
@@ -44,6 +45,11 @@ export class DeployWorkspaceProjectConfirmStep extends AzureWizardPromptStep<Dep
             outputMessage = localize('overwriteConfirmed', 'User confirmed overwrite of container app "{0}" and the latest image of registry "{1}".', context.containerApp?.name, context.registry?.name);
         }
 
+        if (this.hasUnsupportedFeatures(context)) {
+            confirmMessage += '\n\n' + this.unsupportedFeaturesWarning;
+            outputMessage += ' ' + localize('unsupportedOverwriteConfirmed', 'User also confirmed that proceeding means that any unsupported container app features in VS Code will be lost.');
```

Comment:
May be just me, but this feels a little extra.

Reply:
I'll remove it :P